platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/DefaultConfigurationCache.kt

```diff
@@ -381,7 +382,9 @@ class DefaultConfigurationCache internal constructor(
                 // No entry file -> treat the entry as empty/missing/invalid
                 CheckedFingerprint.NotFound
```

Comment:
I think we should also emit an event here, and, capture this, and display it in the UI as well.

Reply:
A status is good for me. So you'd have the `NOT_FOUND` status and no invalidation reasons? Will there be more outcomes in the future or only the ones you wrote down here?

Reply:
Done

Reply:
> So you'd have the `NOT_FOUND` status and no invalidation reasons? Will there be more outcomes in the future or only the ones you wrote down here?

Sorry, I left out this question. I believe https://github.com/gradle/gradle/pull/29655#discussion_r1668391436 has an answer.